src/components/shared/ManagedTree.js

```diff
@@ -154,9 +143,9 @@ class ManagedTree extends Component<Props, State> {
       <div className="managed-tree">
         <Tree
           {...this.props}
-          isExpanded={item => expanded.has(this.props.getPath(item))}
+          isExpanded={item => expanded.has(item.path)}
           focused={focusedItem}
-          getKey={this.props.getPath}
+          getKey={item => item.path}
```

Comment:
perhaps the biggest change here is that I'm no longer using `getPath`... We might need to bring this back so that the tree updates when we pretty print/black box a source, but it was a nice attempt to cleanup